### DISCUSSION TITLE
【求助】朋友对米黑谣言深信不疑怎么办？

### DISCUSSION THREAD

**Discussion by bigetree** [share121/inter-knot#5479] [其他]
我有一个朋友，他对几乎所有的米黑谣言都深信不疑。包括但不限于米游社60w评论都是玩家冲的，米没了疫情加持就要似，米投资核聚变是资产转移，米非常高傲不听玩家意见也不让玩家有意见。
我知道他是战双老玩家，比较偏心库，但这不代表他可以为了黑米而相信谣言也不代表他可以跟风黑米。他自己没有涉足过任何战场，全是听b站的。
我该怎么劝他别信谣言？

**Comment by bigetree** [comment]
对了，他有提到什么尘白3000人吵赢人机，这事是什么意思？我想知道人机怎么被骂破防。

**Reply by BJ-cmyk** [reply]
尘白？这下看懂了，这就不奇怪了，我还以为这种人只在网上存在呢

**Reply by bigetree** [reply]
能详细讲一下吗？

**Comment by LiuZhy2004** [comment]
一般这种情况没法和他辩驳，每个人都自有一套逻辑，很难打破的。你最好也有意识地别和他聊到这个话题，不然你和他都烦。你毕竟是他现实的朋友，他在网络上如何如何并不能影响你和他的关系。

**Reply by bigetree** [reply]
问题是他在现实生活中也说

**Comment by BJ-cmyk** [comment]
朋友？同学而已！就冲着这种二极管脑子就不该跟他交往，一点分辨能力都没有，脑袋里彻底被互联网的那群杂蛆烙上思想钢印了，既然你跟他解释过了没用，那就早点离开他，对你有好处。作冷处理

**Comment by qeryoupadhjlvnim** [comment]
笑死我了你觉得跟一个中专学历并且尬黑都不会的机器人能说出个啥，避开好了

**Comment by Quatfolia** [comment]
清者自清

**Comment by ztjh** [comment]
要么当面直说 要么不搭理 遵从内心

**Comment by bxzwq** [comment]
可能被社区的某些话术洗脑了，一点辨别社区真假的能力都没有，如果怎么劝都没用，那我还是建议你断交吧

**Reply by zouhaoya** [reply]
+1

**Comment by ConsistDust3838** [comment]
这种情况最好还是先好好想想你这个玩米的在他眼里是个什么形象，他能这么随意地信对米哈游的节奏那你在他眼里是不是米孝子呢

**Comment by kittyll1** [comment]
咋劝都没用建议断交
